magefiles/magefile.go

```diff
@@ -496,6 +496,15 @@ func (ci CI) setRequiredEnvVars() error {
 			if !strings.Contains(jobName, "rehearse") {
 				os.Setenv(fmt.Sprintf("%s_CATALOG_URL", envVarPrefix), fmt.Sprintf("https://github.com/%s/%s", pr.Organization, pr.RepoName))
 				os.Setenv(fmt.Sprintf("%s_CATALOG_REVISION", envVarPrefix), pr.CommitSHA)
+				os.Setenv("RELEASE_MANAGED_WORKSPACE", constants.ManagedReleaseTeam)
+				os.Setenv("RELEASE_DEV_WORKSPACE", constants.DevReleaseTeam)
+				if os.Getenv("REL_IMAGE_CONTROLLER_QUAY_ORG") != "" {
+					os.Setenv("IMAGE_CONTROLLER_QUAY_ORG", os.Getenv("REL_IMAGE_CONTROLLER_QUAY_ORG"))
+				}
+
+				if os.Getenv("REL_IMAGE_CONTROLLER_QUAY_TOKEN") != "" {
```

Comment:
what is setting this?

Reply:
It's used in the build PR of push_to_external_registry test case.

Reply:
Then why is it here instead of in the release files? My understanding is the only reason to put something in the magefile is if it needs to be set with variables related to the PR

Reply:
Because the values of these variables may change when we run the test cases manually.

Reply:
If we want to set them and not affect other tests, why don't we set them in the release pipelines suite files?

Reply:
I tried to set it in release files. But the case can't work well because the IMAGE_CONTROLLER_QUAY_ORG   is used in "magefiles/installation/install.go" file.  It seems that the value need to be set since the cluster is installed.